scipy/sparse/linalg/_isolve/symmlq.py

```diff
@@ -0,0 +1,239 @@
+import numpy as np
+from .utils import make_system
+
+
+__all__ = ['symmlq']
+
+
+def symmlq(A, b, x0=None, tol=1e-5, maxiter=None, M=None, callback=None,
+           atol=None, haptol=1e-18, show=False):
+    """
+    Use Symmetric LQ iteration to solve ``Ax = b``.
+
+    Parameters
+    ----------
+    A : {sparse matrix, ndarray, LinearOperator}
+        The real or complex N-by-N matrix of the linear system.
+        Alternatively, `A` can be a linear operator which can
+        produce ``Ax`` using, e.g.,
+        `scipy.sparse.linalg.LinearOperator`.
+    b : {ndarray}
+        Right hand side of the linear system. Has shape (N,) or (N, 1).
+    x0 : {ndarray}
+        Starting guess for the solution.
+    tol, atol : float, optional
+        Tolerances for convergence, ``norm(b - Ax) < max(tol*norm(b), atol)``.
+        The default for `tol` is 1.0e-5.
+        The default for `atol` is ``tol * norm(b)``.
+
+        .. warning::
+
+           The default value for `atol` will be changed in a future release.
+           For future compatibility, specify `atol` explicitly.
+    haptol : float, optional
+        Tolerance for happy breakdown, ``abs(np.inner(r.conjugate(), Mr)) <
+        haptol``.
+        Default is 1.0e-18.
+    maxiter : int, optional
+        Maximum number of iterations.  Iteration will stop after maxiter
+        steps even if the specified tolerance has not been achieved.
+        Default is ``min(10000, n * 10)``, where ``n = A.shape[0]``.
+    M : {sparse matrix, ndarray, LinearOperator}
+        Inverse of the preconditioner of A. M should approximate the
+        inverse of A and be easy to solve for (see Notes). Effective
+        preconditioning dramatically improves the rate of convergence,
+        which implies that fewer iterations are needed to reach a given
+        error tolerance. By default, no preconditioner is used.
+    callback : function, optional
+        User-supplied function to call after each iteration. It is called
+        as `callback(rnorm)`, where `rnorm` is the current preconditioned
+        residual norm.
+    show : bool, optional
```

Comment:
> This should either be `verbose` or `disp` to match the rest of the functions but I am not a big fan of these switches. Etiher shows only relevant info or doesn't show anything. We have many function arguments like this waiting to be deprecated because it didn't serve the purpose.

There are too many files are using the keyword `show`  in `scipy/sparse/linalg/_isolve/` (such as `minres.py`, `lsmr.py`, `lsqr.py`, `tfqmr.py` and some test files in `tests/`). If it's really necessary, I think I can submit a new PR to modify it together for interface consistency.